src/main/java/org/candlepin/audit/ActivationListener.java

```diff
@@ -45,10 +46,15 @@ public ActivationListener(SubscriptionServiceAdapter subService,
     public void onEvent(Event event) {
         if (event.getType().equals(Event.Type.CREATED) &&
             event.getTarget().equals(Event.Target.POOL)) {
+            JsonNode subscriptionId = null;
             try {
-                String subscriptionId = mapper.readTree(event.getEventData())
-                    .get("subscriptionId").asText();
-                subscriptionService.sendActivationEmail(subscriptionId);
+                if (event.getEventData() != null) {
+                    subscriptionId = mapper.readTree(event.getEventData())
+                        .get("subscriptionId");
+                }
```

Comment:
Add some whitespace between these two blocks so it doesn't look like an if/else-if at a glance.